be/src/exec/exchange_node.cpp

```diff
@@ -71,7 +71,7 @@ Status ExchangeNode::prepare(RuntimeState* state) {
     _stream_recvr = state->exec_env()->stream_mgr()->create_recvr(
             state, _input_row_desc, state->fragment_instance_id(), _id, _num_senders,
             config::exchg_node_buffer_size_bytes, _runtime_profile, _is_merging, _sub_plan_query_statistics_recvr,
-            false, false);
+            false, -1, false);
```

Comment:
add some comment to explain why -1 is used here

Reply:
ok